ses

```diff
@@ -130,6 +130,25 @@ if [ -d /etc/salt ]; then
 fi
 
 #############################################################
+section_header "NFS-Ganesha config"
+
+if [ -e /etc/ganesha/ganesha.conf ]; then
+    mkdir -p $LOG/ceph/ganesha/
+    cp /etc/ganesha/ganesha.conf $LOG/ceph/ganesha/ganesha.conf
+    plugin_message "NFS-Ganesha config copied to ceph/ganesha/ subdirectory"
+fi
+
+#############################################################
+section_header "NFS-Ganesha log files"
+
+if [ -e /var/log/ganesha.log ]; then
+    mkdir -p $LOG/ceph/ganesha/log
+    cp /var/log/ganesha.log $LOG/ceph/ganesha/log
```

Comment:
Is there a directory structure also for conf files? I added $LOG/ceph/ganesha/. Should it be $LOG/ceph/conf/ganesha?